validation/exclusions_test.go

```diff
@@ -37,20 +37,36 @@ var exclusions = map[string]map[uint64]bool{
 	ChainIDRPCTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+		12345:    true, // sepolia/opdeployer
 	},
 	GenesisRPCTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+		12345:    true, // sepolia/opdeployer
 	},
 	UniquenessTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   Not in https://github.com/ethereum-lists/chains
 		11763072: true, // sepolia-dev-0/base-devnet-0     Not in https://github.com/ethereum-lists/chains
+		12345:    true, // sepolia/opdeployer
 	},
 
 	// Standard Checks
 	OptimismPortal2ParamsTest: {
 		11763072: true, // sepolia-dev0/base-devnet-0
 	},
+
+	GasTokenTest: {
+		12345: true, // sepolia/opdeployer
+	},
+	L2SecurityConfigTest: {
+		12345: true, // sepolia/opdeployer
+	},
+	DataAvailabilityTypeTest: {
+		12345: true, // sepolia/opdeployer
+	},
+	GPOParamsTest: {
```

Comment:
skipped this because of error:

```
=== FAIL: github.com/ethereum-optimism/superchain-registry/validation TestValidation/op-deploy-sepolia-test_(12345)/GPO_Params (3.08s)
    gpo-params_test.go:41: 
                Error Trace:    /Users/blaine/code/superchain-registry/validation/gpo-params_test.go:41
                                                        /Users/blaine/code/superchain-registry/validation/gpo-params_test.go:53
                                                        /Users/blaine/code/superchain-registry/validation/gpo-params_test.go:63
                                                        /Users/blaine/code/superchain-registry/validation/validation_test.go:43
                Error:          Received unexpected error:
                                0x420000000000000000000000000000000000000F.Decimals(): operation failed permanently after 3 attempts: Post "http://awe.some.rpc": dial tcp: lookup awe.some.rpc: no such host
                Test:           TestValidation/op-deploy-sepolia-test_(12345)/GPO_Params
```